ViewLayer/SongPlayerView/SongPlayerView.cpp

```diff
@@ -23,34 +27,31 @@ void SongPlayerView::handlePlayButtonClicked()
     songPlayer_.togglePlayback();
 }
 
-void SongPlayerView::handleOpenButtonClicked()
-{
-    songPlayer_.openFile();
-}
-
 void SongPlayerView::handleNextButtonClicked()
 {
+    ui_.PlayButton().setChecked(true);
     songPlayer_.playNext();
 }
 
 void SongPlayerView::handlePrevButtonClicked()
 {
+    ui_.PlayButton().setChecked(true);
     songPlayer_.playPrevious();
 }
 
-void SongPlayerView::updateTitle(const QString& filePath)
+void SongPlayerView::handleVolumeControl()
 {
-    ui_.infoLabel().setText(QFileInfo(filePath).fileName());
+    int volume = ui_.volumeControl().value();
+    songPlayer_.adjustVolume(volume);
 }
 
-void SongPlayerView::updatePosition(qint64 position)
+void SongPlayerView::updateGUI(const QString& title, const QString& artist, const QImage& cover)
```

Comment:
We are not using a cover image right now, so currently the cover parameter isn't used. Lets remove it for now until we implement that feature.